### PR TITLE
master-qa-126603

### PR DESCRIPTION
**Pull request by chaowang-123**
https://issues.liferay.com/browse/LPS-126603
@david-gutierrez-mesa 
@liulukiwi
@Tim-Cao

### CODE REVIEW

**Comment by chaowang-123** [comment]
ci:test:sf

**Comment by chaowang-123** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 1]
```diff
@@ -324,6 +324,132 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-96795. content author no content's update permission, having permissions to Update Page Content so could only go to the edit mode."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:327]: Workflow for automation creation:

1. Read the description of story to understand the introduced feature
2. Learn the test points of case
3. Manual testing
4. Write down the manual testing steps
5. Create automation test according to manual testing steps

**Comment by Tim-Cao** [comment]
Hi @chaowang-123, could you following the workflow to double check your test first?

**Comment by Tim-Cao** [comment]
ci:stop

**Comment by liulukiwi** [comment]
pr review in https://github.com/liulukiwi/liferay-portal/pull/345

**Pull request closed without merging**